### PR TITLE
Some more feature page changes

### PR DESCRIPTION
**Pull request by texnixe**

### CODE REVIEW

File: site/templates/features.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <section class="features-section">
         <?= snippet('features/header', [
           'heading' => 'The Panel',
-          'subheading' => 'A control room that feels like home'
+          'subheading' => 'A control room where you will instantly feel at home'
```
Comment (distantnative) [inline site/templates/features.php:12]: Personally, here I like Bastian's original line better. Kurz und knack :D

File: site/templates/features.php [hunk 2]
```diff
@@ -21,23 +21,23 @@
...
-              Prepare your content - pages, articles, events, products - and only publish them when everything is in place with the click of a button. Send preview links to others so they can review your drafts even before they go live.
+              Prepare your content—pages, articles, events, products—and only publish them when everything is in place with the click of a button. Send preview links to others so they can review your drafts before they go live.
             '
           ]) ?>
 
           <?= snippet('features/text', [
             'heading' => 'Drag & Drop sorting',
             'text' => '
-              Sorting shouldn\'t be complicated. Just pick your pages or files up and drop them where you want them to be. Why should it be anymore complicated than that?
+              Sorting pages or files is a breeze. Just pick them up and drop them where you want them to be. It shouldn\'t be more complicated than that?
```
Comment (distantnative) [inline site/templates/features.php:40]: Keeping the question mark at the end intended?

File: site/templates/features.php [hunk 3]
```diff
@@ -365,11 +365,11 @@
           ]) ?>
           <?= snippet('features/text', [
             'heading' => 'KirbyTags',
-            'text'    => 'When Markdown just isn\'t enough, add your own KirbyTags to simplify plain text formatting and injection of complex custom components into long-form text.'
+            'text'    => 'When Markdown isn\'t enough, add your own KirbyTags to simplify plain text formatting and to inject complex custom components into long-form text.'
           ]) ?>
           <?= snippet('features/text', [
             'heading' => 'Caching',
-            'text'    => 'Kirby comes with mighty caching on board. Not fitting for your project? Add the cache driver on your choice and reduce page loads to the blink of an eye.'
+            'text'    => 'Kirby comes with mighty caching on board. Not fitting for your project? Add your cache driver of choice and reduce page loads in the blink of an eye.'
```
Comment (distantnative) [inline site/templates/features.php:372]: Here I meant that the page load only takes a blink of an eye. Not the implementation of the cache driver. That would be a bit exaggerated.

**Comment by distantnative** [comment]
Great! I really like the changes. Only have a few comments/questions.

File: site/templates/features.php (hunk 1 above)
Comment (texnixe) [reply site/templates/features.php:12]: Maybe we can find something better, but feels like home sounds strange to me in combination with a control room.

File: site/templates/features.php (hunk 2 above)
Comment (texnixe) [reply site/templates/features.php:40]: I actually think the whole question is redundant and should be replaced with something better. I didn't have a good idea yet.

File: site/templates/features.php (hunk 3 above)
Comment (texnixe) [reply site/templates/features.php:372]: I've never seen the prepositon "to" in combination with "blink of an eye". Maybe "reduce page load massively".

Comment (bastianallgeier) [reply site/templates/features.php:372]: How about "reduce page loads significantly"

File: site/templates/features.php (hunk 1 above)
Comment (bastianallgeier) [reply site/templates/features.php:12]: Maybe we should remove the "control room" part entirely and make it more human friendly.

**Comment by bastianallgeier** [comment]
All changes are now in my branch again, but without the /product URLs. Let's move on from there.

File: site/templates/features.php (hunk 3 above)
Comment (texnixe) [reply site/templates/features.php:372]: That sounds much better.

**Pull request closed without merging**